### ISSUE TITLE
Are there any issues in GFL that should be done before publishing this sim?

### ISSUE THREAD

**Issue by zepumph** [phetsims/gravity-force-lab-basics#153]
@terracoda does it look like any issues in https://github.com/phetsims/gravity-force-lab/issues block GFLB? I found a few in inverse square law that did, so I thought I would create an issue here for review. I'll so the same look through.

**Comment by terracoda** [comment]
**These don't seem to be blocking:**
- https://github.com/phetsims/gravity-force-lab/issues/150 - the times symbol is not in GFLB
- https://github.com/phetsims/gravity-force-lab/issues/149 - is no longer relevant. We are going to use "force vectors" everywhere.
- https://github.com/phetsims/gravity-force-lab/issues/141 - about ruler, so not relevant
- https://github.com/phetsims/gravity-force-lab/issues/140 - about ruler, not relevant
- https://github.com/phetsims/gravity-force-lab/issues/139 - about ruler, not relevant
- https://github.com/phetsims/gravity-force-lab/issues/138 - about ruler, not relevant
- https://github.com/phetsims/gravity-force-lab/issues/133 - no rounding is needed in GFLB, so I think this is not relevant
- https://github.com/phetsims/gravity-force-lab/issues/110 - no checkbox like this in GFLB, so not relevant
- https://github.com/phetsims/gravity-force-lab/issues/104 - about ruler, so not relevant
- https://github.com/phetsims/gravity-force-lab/issues/48 - You've done a code review for GFLB, so likely not relevant

**Not sure about these ones:**
- https://github.com/phetsims/gravity-force-lab/issues/137 - we will be adopting the approach from GFLB, so not sure if this needs to be addressed or not? What do you think @zepumph?
- https://github.com/phetsims/gravity-force-lab/issues/144 - The dialog in GFLB does indeed overlap the black navbar. @zepumph, is it possible to reduce some white space above the Basic Actions heading in the GFLB's dialog to address this issue?
- https://github.com/phetsims/gravity-force-lab/issues/102 - Check this one about the credits. I don't no who besides the A11y folks need to be added to GFLB.
- https://github.com/phetsims/gravity-force-lab/issues/97 - About teacher docs, checking with @arouinfar  

I think that's it.

**Comment by terracoda** [comment]
@zepumph, in my opinion, I don't think either https://github.com/phetsims/gravity-force-lab/issues/137 or https://github.com/phetsims/gravity-force-lab/issues/144 should block anything for GFLB.
